src/lib/stitching/gravity/schema.ts

```diff
@@ -50,7 +52,7 @@ export const executableGravitySchema = () => {
     // We have the same restrictions for root, so let's prefix
     // for now
     new RenameRootFields((type, name, _field) => {
-      if (type === "Query") {
+      if (type === "Query" && !allowList.includes(name)) {
```

Comment:
@zephraph this is the change that we'd like to get some advice on. We added a new `viewingRoom` type to the gravity schema, and tried to query for it in Metaphysics' graphiql. What we found instead in the Docs tab was an `_unused_gravity_viewingRoom` type.

We were hoping that you may be able to shine some light on the origins of `_unused_gravity_` being prepended to types in the gravity schema, so that we can determine if this change makes sense.

Reply:
I’m trying to dig up more context. I believe this was to a aid in cleanup of unused fields by giving something for a transformer to hook off of. 

I’ll get back with you once I have some context.

Reply:
thanks! much appreciated 🙏

Reply:
Okay, while I haven't found the exact situation in which it was added, it was done as a part of https://github.com/artsy/metaphysics/pull/1854.

One of the things @alloy was trying to do generally is to clean up the root fields that were in the schema. _Especially_ root fields that weren't being used. Gravity actually had a lot of those. There was a transformer that went through and removed all the `_unused_gravity_*` fields from the final schema.

Reply:
Correct; iirc you could probably just drop all of them when they are stitched in, as long as at least 1 type remains–which it sounds like you now have (?).

Reply:
Thanks for digging into that. Why is it problematic for those unused root fields to be in the schema?

Reply:
Our [schema design](https://github.com/artsy/README/blob/0666a71997b5210315f70a34971f52c786ff8f53/playbooks/graphql-schema-design.md#root-fields) document talks about this a bit. 

Root fields are the entry points into our data and we want to reduce the overall number of entrypoints to high level domain objects. This is mostly to help keep the overall complexity of our schema down. We definitely don't want to add things that aren't being used. When you add something someone is likely to use it down the road. When you decide that the thing you added isn't right for the original problem, then you either have to add something new or change the original one... and depending on who is using it, that can get problematic. 

To boil it all down, schema designs can be hard to change later. Let's only build out what we need and only when we need it.

Reply:
I love the idea of curating the high level domain objects that we expose as entry-points. That makes a lot of sense to me.